document/src/vespa/document/fieldvalue/serializablearray.cpp

```diff
@@ -26,20 +26,19 @@ class BufferMap : public BufferMapT {
 
 }
 
-SerializableArray::SerializableArray(EntryMap entries, ByteBuffer buffer,
-                                     CompressionConfig::Type comp_type, uint32_t uncompressed_length)
-    : _entries(std::move(entries)),
-      _uncompSerData(),
-      _unlikely()
+void
+SerializableArray::set(EntryMap entries, ByteBuffer buffer,
+                       CompressionConfig::Type comp_type, uint32_t uncompressed_length)
 {
-
+    _entries = std::move(entries);
     if (CompressionConfig::isCompressed(comp_type)) {
         _unlikely = std::make_unique<RarelyUsedBuffers>();
         _unlikely->_compSerData = std::move(buffer);
         _unlikely->_serializedCompression = comp_type;
         _unlikely->_uncompressedLength = uncompressed_length;
```

Comment:
Should `_uncompSerData` also be reset in this branch?

Reply:
Yes, it should, even if the place this method is used, this is under control.